apisix/plugins/kafka-logger.lua

```diff
@@ -60,7 +59,8 @@ local schema = {
         buffer_duration = {type = "integer", minimum = 1, default = 60},
         inactive_timeout = {type = "integer", minimum = 1, default = 5},
         batch_max_size = {type = "integer", minimum = 1, default = 1000},
-        include_req_body = {type = "boolean", default = false}
+        include_req_body = {type = "boolean", default = false},
+        cluster_name = {type = "integer", default = 1},
```

Comment:
Do you need to limit the minimum value?

Reply:
It confuses me, why name is an integer?

Reply:
> It confuses me, why name is an integer?

follow the origin design: https://github.com/doujiang24/lua-resty-kafka#new-1

```
The third optional cluster_name specifies the name of the cluster, default 1 (yeah, it's number). You can Specifies different names when you have two or more kafka clusters. And this only works with async producer_type.
```

I think using numbers should satisfy the need.

Reply:
Could you add it as a comment?

Reply:
Should we highlight that this is a design in the documents?

Reply:
done